ipmi/test_ipmi_shell_inband_user_account.robot

```diff
@@ -28,16 +28,25 @@ Test Host IPMI Inband Add User Account
     ...     M21_IPMI_1_IB_Add_User_Account
 
     ${random_username}=  Generate Random String  8  [LETTERS]
-    ${random_userid}=  Evaluate  random.randint(3, 15)  modules=random
+    ${random_userid}=  Evaluate  random.randint(3, 10)  modules=random
+
+    # generate a random password with atleast one uppercase, one lowercase and a numeral
+    ${uppercase}=  Generate Random String  4  [UPPER]
```

Comment:
Let's use 16 bytes password, 6 upper + 6 lower + 4 number. In case, there is minimum length requirement.

Reply:
Done.